services/web/server/src/simcore_service_webserver/resource_manager/garbage_collector.py

```diff
@@ -153,12 +213,24 @@ async def remove_resources_if_guest_user(
     logger.debug(
         "Removing project '%s' from the database", project_uuid,
     )
-    try:
-        await delete_project_from_db(app, project_uuid, user_id)
-    except ProjectNotFoundError:
-        logging.warning("Project '%s' not found, skipping removal", project_uuid)
 
-    await delete_user(app, user_id)
+    if disable_access_check:
+        await garbage_collector_remove_project_from_db(
+            app=app, project_uuid=project_uuid
+        )
+    else:
+        try:
+            await delete_project_from_db(app, project_uuid, user_id)
+        except ProjectNotFoundError:
+            logging.warning("Project '%s' not found, skipping removal", project_uuid)
```

Comment:
not sure I get that completely. the function delete_project_from_db deletes entry in the project,comp_pipeline and the data folders (also I see this seems to happen quite a lot everywhere). and with your special function you only delete the entry in the project table?

Reply:
That is actually a BUG. Need to fix it

Reply:
And the `garbage_collector_remove_project_from_db`  is not enough to properly clean up

Reply:
this was fixed after the merge, the bugged `garbage_collector_remove_project_from_db ` is no longer needed